stories/features/report-log.fmf

```diff
@@ -0,0 +1,25 @@
+title: Save a log file.
```

Comment:
```suggestion
title: Save a log file
```

Reply:
Removed full stop character in commit [[249c948](https://github.com/teemtee/tmt/pull/1342/commits/249c948bfd797ece4b43eb89de5a4d033f428dca)]